### PR TITLE
updated io_utils.py

### PR DESCRIPTION
**Pull request by haripriya9647**
## Description
adding inline documentation to the files in utilities folder
## Relevant Issue
Issue #269  the issue here

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/NimbleEdge/EnvisEdge/labels)
- [x] My changes are covered by tests

### CODE REVIEW

File: fedrec/utilities/io_utils.py [hunk 1]
```diff
@@ -67,6 +89,9 @@ def dash_separated_ints(value):
 
 
 def dash_separated_floats(value):
+    """ 
+    Returns dash seperated list of float values.
```
Comment (divyanshugit) [inline fedrec/utilities/io_utils.py:93]: You can re-write it as
```suggestion
    This method takes a list of float values as input and returns the dash-separated list of values.   
```
You can add an example for it as well. and check the pep8 as well in case it exceeds the value.

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/utilities/io_utils.py [hunk 2]
```diff
@@ -67,6 +90,17 @@ def dash_separated_ints(value):
...
 def dash_separated_floats(value):
+    """ 
+    This method takes a list of float values as input 
+    and returns the dash-seperated list of values.
+
+    Example
+    -------
+    value = "1.0-1.0"
+    x = value.split("-")
+    print(x)
```
Comment (divyanshugit) [inline fedrec/utilities/io_utils.py:101]:
```suggestion
    >>> value = "1.0-1.0"
    >>> x = value.split("-")
    >>> print(x)
    ['1.0', '1.0']

```

File: fedrec/utilities/io_utils.py [hunk 3]
```diff
@@ -54,6 +73,10 @@ def dictify(iterable):
 
 
 def dash_separated_ints(value):
+    """ 
+    This method takes a list of int as input 
+    and returns the dash-seperated list of values.
+    """
```
Comment (divyanshugit) [inline fedrec/utilities/io_utils.py:79]: Add a example here as well

File: fedrec/utilities/io_utils.py [hunk 4]
```diff
@@ -30,6 +43,12 @@ def to_dict_with_set_values(d):
 
 
 def save_tensors(tensors, path) -> str:
+    """
+    Checks if the file path is true then it returns 
+    the tensor path else the path is joined to
+    "completeName" which is further returned.
+
```
Comment (divyanshugit) [inline fedrec/utilities/io_utils.py:50]: Add an example over as well

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by haripriya9647** [comment]
Working on remaining suggested commits @divyanshugit

File: fedrec/utilities/cuda_utils.py [hunk 5]
```diff
@@ -4,6 +4,13 @@
 
 
 def map_to_cuda(args, device=None, **kwargs):
+    """
+    CUDA differentiates between the host, the computer's CPU and
+    main memory and device, the GPU. In CUDA programs, functions
+    specify both on which device they are run and on which devices
+    they are callable from.In this method the processes are being
+    mapped to cuda.
```
Comment (divyanshugit) [inline fedrec/utilities/cuda_utils.py:12]: You can modify the comment a bit. At first you should give information about what is happening then explain it.

File: fedrec/utilities/cuda_utils.py [hunk 6]
```diff
@@ -15,12 +22,18 @@ def map_to_cuda(args, device=None, **kwargs):
 
 
 def map_to_list(model_params):
+    """
+    Cuda processes are being mapped here to lists.
```
Comment (divyanshugit) [inline fedrec/utilities/cuda_utils.py:26]: Here model parameters are being mapped to the list. You can explain a bit more about it

File: fedrec/utilities/cuda_utils.py [hunk 7]
```diff
@@ -15,12 +22,18 @@ def map_to_cuda(args, device=None, **kwargs):
...
+    """
     for k in model_params.keys():
         model_params[k] = model_params[k].detach().numpy().tolist()
     return model_params
 
 
 def mapping_processes_to_gpus(gpu_config, process_id, worker_number):
+    """
+    Processes are being mapped to gpus.
```
Comment (divyanshugit) [inline fedrec/utilities/cuda_utils.py:35]: Same here.

File: fedrec/utilities/io_utils.py [hunk 8]
```diff
@@ -5,6 +5,11 @@
 
 
 def load_tensors(path):
+   """
+   Checks if the file path is true, if yes then it
+   loads the path to tensor and returns it else raises
+   a value error if the path doesn't exist.
+   """
```
Comment (divyanshugit) [inline fedrec/utilities/io_utils.py:12]: Can you please re-write this whole thing?

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by divyanshugit** [comment]
LGTM!! @ramesht007

**Review by ramesht007 — APPROVED** [review]
LGTM !!
